package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-offline-edge-lambda",
-  "version": "1.2.0",
```

Comment:
Is this manually incremented or should this be put back to 1.2.0 for an auto increment?